src/System.Windows.Forms/src/System/Windows/Forms/PropertyGridInternal/PropertyGridView.cs

```diff
@@ -772,9 +772,12 @@ public PropertyGridView(IServiceProvider serviceProvider, PropertyGrid propertyG
             return GetAllGridEntries();
         }
 
-        internal Rectangle AccessibilityGetGridEntryBounds(GridEntry gridEntry) {
+        internal Rectangle AccessibilityGetGridEntryBounds(GridEntry gridEntry)
+        {
             int row = GetRowFromGridEntry(gridEntry);
-            if (row == -1) {
+
+            if (row < 0)
```

Comment:
When do we get negative row indices other than -1?

Reply:
```suggestion
            int row = GetRowFromGridEntry(gridEntry);
```
This method returns -1 in case entry == null. If the current entity is higher than the first visible entity, then this method returns the negative index. This way I fixed the incorrect display of bounds above the visible area.

Reply:
Thank you for the explanation